python/ray/serve/_private/constants.py

```diff
@@ -36,8 +36,13 @@
 #: Max concurrency
 ASYNC_CONCURRENCY = int(1e6)
 
-# How often to call the control loop on the controller.
-CONTROL_LOOP_PERIOD_S = 0.1
+# How long to sleep between control loop cycles on the controller.
+CONTROL_LOOP_INTERVAL_S = float(os.getenv("RAY_SERVE_CONTROL_LOOP_INTERVAL_S", 0.1))
```

Comment:
I thought `INTERVAL` made more sense than `PERIOD` as the name, since it's the time *between* cycles, not a target for when the next cycle starts.

Reply:
way better :)